new-log-viewer/src/utils/file.ts

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable @stylistic/js/array-element-newline */
+const SI_UNITS = ["B", "kB", "MB", "GB", "TB", "PB", "EB", "ZB", "YB", "RB", "QB"];
+const IEC_UNITS = ["B", "KiB", "MiB", "GiB", "TiB", "PiB", "EiB", "ZiB", "YiB", "RiB", "QiB"];
+/* eslint-enable @stylistic/js/array-element-newline */
+
+/**
+ * Formats the given size value using either SI (kB, MB, etc.) or IEC (KiB, MiB, etc.) units.
+ *
+ * @param value The input value in bytes.
+ * @param useSiUnits Whether to use SI units.
+ * @param numFractionalDigits Number of digits to keep after the decimal point.
+ * @return The value formatted as "<value> <unit>".
+ */
+const formatSizeInBytes = (
+    value: number,
+    useSiUnits: boolean = true,
+    numFractionalDigits: number = 1
+): string => {
+    const units = useSiUnits ?
+        SI_UNITS :
+        IEC_UNITS;
+    /* eslint-disable no-magic-numbers */
```

Comment:
```suggestion
```
Numbers 10 and 3 have been added into an exception list in `eslint-config-yscope` 0.0.30 or above: https://github.com/y-scope/eslint-config-yscope/commit/515b6dd15daae18a08c85e5e7a147159cf5b6216

Sorry for the confusion.